contracts/TokenVault.sol

```diff
@@ -178,6 +189,35 @@ contract TokenVault is Ownable, Recoverable {
     return token.balanceOf(address(this));
   }
 
+  /// @dev Check how many tokens "investor" can claim
+  /// @param investor Address of the investor
+  /// @return uint How many tokens the investor can claim now
+  function getClaimableAmount(address investor) public constant returns (uint claimableAmount) {
+    uint maxTokensLeft = balances[investor] - claimed[investor];
+    uint sinceLastClaim = lastClaimedAt[investor];
```

Comment:
`sinceLastClaim` is a bad name too because `since...` to me signifies `secondsSince`, but this holds a timestamp!